src/components/EventDetails.js

```diff
@@ -0,0 +1,111 @@
+
+import React, { Component } from 'react';
+import Moment from 'moment';
+import { useParams } from 'react-router-dom';
+import '../css files/EventDetails.css'
+import { withRouter } from "react-router";
+
+
+
+class EventDetails extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            people: []
+        }
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    componentDidMount() {
+        const eventId = this.props.match.params.eventId
+        fetch(`http://127.0.0.1:3001/people/${eventId}`, {
+            method: 'GET'
+        })
+            .then(response => response.json())
```

Comment:
We could isolate this functionality (i.e. the actual making of the fetch request and the extraction of response.json() into a separate function, ideally in a separate file (along with the other fetch request) - why? Separation of concerns.  This is not explicitly necessary in an app like this that makes use of only two different fetch requests, but in the bigger-picture view isolating the API calls to a separate file so that the client can import just the necessary methods by name and use them, leaving their code safely in a different location.  

This is a bit like the Design Pattern called [Facade](https://sourcemaking.com/design_patterns/facade) - the "subsystem" of making API calls and response handling is encapsulated in a separate interface that simplifies its use.